src/core/public/utils/events.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/** @internal */
+export const KBN_LOAD_MARKS = 'kbnLoad';
+
+export { KIBANA_LOADED_EVENT } from '../../utils';
+
+export const LOAD_START = 'load-started';
+export const LOAD_BOOTSTRAP_START = 'bootstrap-started';
+export const LOAD_CORE_CREATED = 'core-created';
+export const LOAD_SETUP_DONE = 'setup-done';
+export const LOAD_START_DONE = 'start-done';
+export const LOAD_FIRST_NAV = 'first-app-nav';
+
+export interface PerformanceMetricEvent {
+  duration?: number;
+  jsHeapSizeLimit?: number;
+  totalJSHeapSize?: number;
+  usedJSHeapSize?: number;
+  key1?: string;
+  value1?: number;
+  key2?: string;
+  value2?: number;
+  key3?: string;
+  value3?: number;
+  key4?: string;
+  value4?: number;
+  key5?: string;
+  value5?: number;
+}
+
+export const PERFORMANCE_METRIC_EVENT_SCHEMA: Record<keyof PerformanceMetricEvent, any> = {
```

Comment:
With the new "everything is a package" initiative, we are moving `core` to packages. I think we can create a new package with this type of performance helpers instead of keeping them in `core/utils`. What do you think?

Reply:
Done